doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,174 @@
+"""
+=====================================================================
+Title
```

Comment:
We also suggested that @etpeterson would help @RafaelNH in writing this tutorial so he can be accustomed to the DIPY development processes. I hope this is still on the horizon. Let me know otherwise.